### PR TITLE
Fix: Error in Toggle, Accordion icons and active inline-font-awesome experiment [ED-4348,ED-4436]

### PR DESCRIPTION
**Pull request by rotemee**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by udidol — APPROVED** [review]

File: includes/managers/icons.php [hunk 1]
```diff
@@ -256,6 +256,12 @@ public static function get_font_icon_svg( $icon, $attributes = [] ) {
 			self::$font_icon_svg_symbols[ $icon_data['key'] ] = $icon_data;
 		}
 
+		if ( array_key_exists( 'class', $attributes ) && ! is_array( $attributes['class'] ) ) {
```
Comment (kobizz) [inline includes/managers/icons.php:259]:
```suggestion
		if ( ! empty( $attributes['class'] ) && ! is_array( $attributes['class'] ) ) {
```

File: includes/managers/icons.php [hunk 2]
```diff
@@ -256,6 +256,12 @@ public static function get_font_icon_svg( $icon, $attributes = [] ) {
...
+			$class_name = $attributes['class'];
+
+			$attributes['class'] = [ $class_name ];
```
Comment (kobizz) [inline includes/managers/icons.php:262]: You can do it in one line: 
```suggestion
			$attributes['class'] = [ $attributes['class'] ];
```

File: includes/widgets/accordion.php [hunk 3]
```diff
@@ -299,6 +299,7 @@ protected function register_controls() {
 				'type' => Controls_Manager::COLOR,
 				'selectors' => [
 					'{{WRAPPER}} .elementor-accordion-icon, {{WRAPPER}} .elementor-accordion-title' => 'color: {{VALUE}};',
+					'{{WRAPPER}} .elementor-accordion-icon svg' => 'fill: {{VALUE}};',
```
Comment (kobizz) [inline includes/widgets/accordion.php:302]: Why don't you use a CSS variable instead?

**Review by kobizz — CHANGES_REQUESTED** [review]

**Comment by kobizz** [comment]
Fix PR Linter and check why is `Elementor Tests / Screenshotter` failing

File: includes/widgets/accordion.php (hunk 3 above)
Comment (rotemee) [reply includes/widgets/accordion.php:302]: As agreed, keeping it as the rest of the file.

File: includes/managers/icons.php (hunk 1 above)
Comment (rotemee) [reply includes/managers/icons.php:259]: Done.

File: includes/managers/icons.php (hunk 2 above)
Comment (rotemee) [reply includes/managers/icons.php:262]: Done.

**Comment by rotemee** [comment]
@kobizz - Please note, in the CSS files of both widgets, I've changed '.e-font-icon-svg' to 'svg' in order to support custom uploaded SVG files.

**Pull request merged**